helpers/regions.js

```diff
@@ -23,7 +25,7 @@ module.exports = {
 	iam: ['us-east-1'],
 	route53: ['us-east-1'],
 	route53domains: ['us-east-1'],
-	s3: ['us-east-1'],
```

Comment:
This is an oddity of the AWS API, but we have to use us-east-1 to query all buckets, even though they're created in other regions.